src/Redux/actions.tsx

```diff
@@ -469,6 +470,173 @@ export const updateAsset = (id: string, params: object) =>
+
+export const searchByHealthId = (healthId: string) =>
+  fireRequest("searchByHealthId", [], { healthId });
+
+export const initiateAbdmAuthentication = (
+  authMethod: string,
+  healthid: string
+) => fireRequest("initiateAbdmAuthentication", [], { authMethod, healthid });
+
+export const confirmWithAadhaarOtp = (
+  txnId: string,
+  otp: string,
+  patientId?: string
+) => fireRequest("confirmWithAadhaarOtp", [], { txnId, otp, patientId });
+
+export const confirmWithMobileOtp = (
+  txnId: string,
+  otp: string,
+  patientId?: string
+) => fireRequest("confirmWithMobileOtp", [], { txnId, otp, patientId });
+
+export const linkViaQR = (abha_details: any, patientId?: string) => {
+  return fireRequest("linkViaQR", [], {
+    ...abha_details,
+    patientId,
+  });
+};
+
+export const linkCareContext = (
+  consultationId: string,
+  data: { name?: string; gender?: "M" | "F" | "O"; dob?: string }
+) => {
+  return fireRequest("linkCareContext", [], {
+    consultation: consultationId,
+    ...data,
+  });
+};
+
+export const getAbhaCard = (patient: string, type: "pdf" | "png") => {
+  return fireRequest("getAbhaCard", [], {
+    patient,
+    type,
+  });
+};
+
+export const getHealthInformation = (artefactId: string) => {
+  return fireRequest("getHealthInformation", [], { artefactId });
+};
+
+export const healthFacilityActions = {
+  list: (params: object) => {
+    return fireRequest("listHealthFacilities", [], params);
+  },
+
+  create: (data: object) => {
+    return fireRequest("createHealthFacility", [], data);
+  },
+
+  read: (id: string) => {
+    return fireRequest(
+      "getHealthFacility",
+      [],
+      {},
+      { facility_id: id },
+      undefined,
+      true
+    );
+  },
+
+  update: (id: string, data: object) => {
+    return fireRequest("updateHealthFacility", [], data, {
+      facility_id: id,
+    });
+  },
+
+  partialUpdate: (id: string, data: object) => {
+    return fireRequest("partialUpdateHealthFacility", [], data, {
+      facility_id: id,
+    });
+  },
+
+  registerService: (id: string) => {
+    return fireRequest(
+      "registerHealthFacilityAsService",
+      [],
+      {},
+      { facility_id: id }
+    );
+  },
+};
+
+export const consentActions = {
+  list: (params: object) => {
+    return fireRequest("listConsents", [], params);
+  },
+
+  create: (data: object) => {
+    return fireRequest("createConsent", [], data);
+  },
+
+  read: (id: string) => {
+    return fireRequest("getConsent", [], {}, { id });
+  },
+
+  status: (id: string) => {
+    return fireRequest("checkConsentStatus", [], {}, { id }, undefined, true);
+  },
+};
+
+export const patientsActions = {
+  find: (params: object) => {
+    return fireRequest("findPatients", [], params);
+  },
+};
+
+export const listAssetAvailability = (params: object) =>
+  fireRequest("listAssetAvailability", [], params);
+export const getAssetAvailability = (id: string) =>
+  fireRequest("getAssetAvailability", [], {}, { id });
+
```

Comment:
Was this added accidentally by a bad resolve of merge conflict?

Also, I see only `consentActions` redux action that is being newly used by useDispatch in this PR. Can we migrate that to `useQuery` instead?

Rest of the newly added redux actions can be removed since they are not referenced nor used by useQuery.